doc/non-ansible-configuration-notes.md

```diff
@@ -115,6 +115,53 @@ Then, manually:
 * Update <https://github.com/nodejs/build/blob/master/ansible/inventory.yml> to reflect any host additions or changes (this is primarily for automatic .ssh/config setup purposes currently)
 * Run Ansible from <https://github.com/nodejs/build/tree/master/setup/raspberry-pi> with `ansible-playbook ansible-playbook.yaml -i ../ansible-inventory --limit <hostname>`
 
+## git on CentOS 5
+
+GitHub have removed TLS 1.0 & 1.1 support, leaving CentOS 5 support in the lurch. Git has to be manually built on CentOS 5, this also requires:
+
+ * Manually fixing yum repos to point to a still-available CentOS 5 repo (many have been removed and the mirror system doesn't work for 5 anymore) so we can install a couple of dependencies
+ * Removing old yum & curl
```

Comment:
From what I see below, shouldn't this be `Removing old git & curl`?

Reply:
thanks @joaocgreis, fixed and force pushed